### PR TITLE
MOB-437: Add templates for issues, pull requests and contributing

### PR DESCRIPTION
**Pull request by mats-claassen**
Fixes issue #3

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Before creating a new issue:
+  * Make sure you are using the [latest release](https://github.com/IDme/ID.me-WebVerify-SDK-iOS/releases).
+  * Check if the issue was [already reported or fixed](https://github.com/IDme/ID.me-WebVerify-SDK-iOS/issues?utf8=%E2%9C%93&q=is%3Aissue). Notice that it may not be released yet.
+  * If you found a match add a brief comment "I have the same problem" or "+1". This helps prioritize the issues addressing the most common and critical ones first. If possible, add additional information to help us reproduce and fix the issue. Please use your best judgement.    
+* Reporting issues:
+  * Please include the following information to help maintainers to fix the problem faster:
+    * Xcode version you are using.
+    * iOS version you are targeting.
+    * Full Xcode console output of stack trace or code compilation error.
+    * Any other additional detail you think it would be useful to understand and solve the problem.
```
Comment (m-revetria) [inline .github/CONTRIBUTING.md:19]: what do you think about requesting the version of the SDK being used?

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+* Add test coverage to the feature or fix. We only accept new feature pull requests that have related test coverage. This allows us to keep the library stable as we move forward.
+* Remember to document the new feature. We do not accept new feature pull requests without its associated documentation.
+* In case of a new feature please update the example project showing the feature.
+* Please only one fix or feature per pull request. This will increase the chances your feature will be merged.
+
+
+###### Suggested git workflow to contribute
+
+1. Fork the XLProductName repository.
```
Comment (m-revetria) [inline .github/CONTRIBUTING.md]: please update the name here

**Review by m-revetria — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (mats-claassen) [reply .github/CONTRIBUTING.md:19]: That is right. It is included in the Issue template

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+6. Stage all your changes `git add .` and commit them `git commit -m "Your commit message"`
+7. Make sure your branch is up to date with upstream master, `git pull --rebase upstream master`, resolve conflicts if necessary. This will move your commit to the top of git stack.
+8. Squash your commits into one commit. `git rebase -i HEAD~6` considering you did 6 commits.
+9. Push your branch into your forked remote repository.
+10. Create a new pull request adding any useful comment.
+
+
+###### Code style and conventions
+
+We try to follow our [swift style guide](https://github.com/github/objective-c-style-guide). Following it is not strictly necessary to contribute and to have a pull request accepted but project maintainers try to follow it.
```
Comment (zachallett) [inline .github/CONTRIBUTING.md]: Link title references swift however is linking to the Obj-c guide

Comment (mats-claassen) [reply .github/CONTRIBUTING.md]: Copy pasting error :)

**Review by m-revetria — APPROVED** [review]

**Pull request merged**